### DISCUSSION TITLE
Does Dkron provides support to store schedule in postgres or any other databases?

### DISCUSSION THREAD

**Discussion by gm7y8** [dkron-io/dkron#1559] [Q&A]

**Comment by vcastellm** [comment]
No, it doesn't support storing to postgres as it is not a distributed store.

Well, key-value stores are a type of Database, and the only supported databases are distributed key-value stores.

**Comment by raghav-nayak** [comment]
Hello Dkron team,

I tried to use the DKron with K8S setup. Everything worked except persistent volume. I checked your helm project. Statefulset(sts) is used instead of deployment. This is ok but each sts pod needs separate PVC. If you scale down, the data stored a particular PVC is not accessible.

With deployment, PVC does not work properly.

With any kind of database, each deployment can share the resources. In cluster mode, leader takes all CRUD operations, there will not be any race conditions too.

Love to know what your thoughts on using any database. If DKron already supports a database as backend, let me know.

- Raghav

**Reply by vcastellm** [reply]
Hi @raghav-nayak, as I mentioned, the storage is embedded KV store, no external database is supported, this is to simplify the design and implement the distributed KV store.

My recommendation is to use the helm chart and statefullsets.
